content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -155,6 +156,7 @@ jobs:
               then
                 fail_site $i
               fi
+            fi
```

Comment:
Getting this error:
```
selected worker: 12e736b474fb
++ pwd
+ CURDIR=/tmp/build/d1fb8792
+ [[ '' != '' ]]
+ GITKEYSSH=
bash: -c: line 54: syntax error near unexpected token `fi'
```

The pipeline completes successfully after removing the new `fi` line, but got an `Invalid BaseURL` error on every site:
```
Error: Error building site: logged 1 error(s)
++ return 1
+ result='Start building sites … 
hugo v0.98.0-165d299cde259c8b801abadc6d3405a229e449f6+extended linux/amd64 BuildDate=2022-04-28T10:23:30Z VendorInfo=gohugoio

ERROR 2022/08/02 15:09:11 Invalid BaseURL: "course-v2/courses/2-mb-site-2-spring-2025"
```